docs/cloud/aws/lca1/customer-ssp/index.md

```diff
@@ -0,0 +1,97 @@
+# System Security Plan (SSP) Information
+
+Systems that fall under the [CU Data Classification](https://www.cu.edu/data-governance/resources-support/data-classification) of **Confidential Information** or
+**Highly Confidential Information** are required to provide a System Security Plan (SSP).
+An SSP is a document that outlines how an organization protects the security of a system.
+An SSP is a critical document that helps organizations meet compliance requirements.
+
+The [OIT IT Security](https://oit.colorado.edu/services/it-security) group is responsible for guiding and evaluating your SSP.
+[OIT IT Security](https://oit.colorado.edu/services/it-security) prescribes the [NIST SP 800-171 Rev. 2](https://csrc.nist.gov/pubs/sp/800/171/r2/upd1/final)
```

Comment:
> Can we point to something on the security site that defines the control set? If not, we need to identify this gap for security - we aren't the authority here and this can change over time.

I looked for a documented statement from IT Security that mentioned this, but did not find a written statement that NIST 800-171 R2 was the official spec.  This has been captured as a gap in the story.  I'll incorporate the link references you added to this section.